source/dump.f90

```diff
@@ -1666,7 +1669,63 @@ subroutine dump_beam_population(nturn, i, ix, unit, fmt, lhighprec, loc_clo, tas
 #endif
     end if
 
+  ! ------------------------------------------------------------------ !
+  !  Format #101                                                       !
+  !  Same as fmt 3, but with additional variable                       !
+  ! ------------------------------------------------------------------ !
+  else if(fmt == 101) then
```

Comment:
Since all the existent formats are in numerical order, why not just use the next number in the line, 10, instead of jumping to 101?